src/main/java/de/uniwue/web/controller/FileController.java

```diff
@@ -126,20 +126,15 @@ private void init() {
 					throw new IOException("File does not exist");
 				imageFile = matchingFiles[0];
 			} else {
-				List<String> supportedImageExt = Arrays.asList(".png", ".jpg", ".jpeg", ".tif", ".tiff");
 				List<String> foundImages = new LinkedList<>();
-				Map<String, String> localImageMap = fileManager.getLocalImageMap();
-				for ( String ext: supportedImageExt) {
-					String imgWithExt = new File(image).getName() + ext;
-					if(localImageMap.containsKey(imgWithExt)) {
-						foundImages.add(localImageMap.get(imgWithExt));
-					}
-				}
+				Map<String, List<String>> localImageMap = fileManager.getLocalImageMap();
 				if(foundImages.size() == 0) {

```

Comment:
In the refactored code `foundImages` will always have a size of `0` at this point. Can be removed.

Reply:
removed with [f0680c1 ](https://github.com/OCR4all/LAREX/pull/273/commits/f0680c1c93de14cef36e45eeeb7901284530f72c)